### PR TITLE
Ontotext suggestions setup

### PR DESCRIPTION
**Pull request by andreic92**
* Integration with another downstream suggester: ontotext-suggestions-api (insert link to gitrepo here);
* Add flags for three types for the moment. Easy to extend to the others as well;

[Jira issue](https://jira.ft.com/browse/FDC-207)

### CODE REVIEW

File: service/suggester.go [hunk 1]
```diff
@@ -167,23 +226,63 @@ func filterOutAuthors(resp SuggestionsResponse) []Suggestion {
...
+		}
+		resp.Suggestions = doConceptsFilteringOut(resp, filter)
+	}
+
+	return resp.Suggestions, nil
 }
 
 func (suggester *FalconSuggester) GetSuggestions(payload []byte, tid string, flags SourceFlags) (SuggestionsResponse, error) {
+	if flags.Debug != "" {
+		log.WithField("Flags", flags.Flags).Info("Ontotext called")
```
Comment (tosan88) [inline service/suggester.go]: _Ontotext called_ ?

**Comment by andreic92** [comment]
From what I can see, `go-logger` is pretty old (0.0.3). Could you update it?
```
[[constraint]]
  name = "github.com/Financial-Times/go-logger"
  version = "1.0.0"
```

**Comment by andreic92** [comment]
Could you change the way ersatz fixtures are loaded? Right now is done in [_ft/hooks.js](https://github.com/Financial-Times/public-suggestions-api/blob/823448575f5c06af1fb297622f1e3a9ee266a62f/_ft/dredd-hooks.js#L7) but there is a prettier way to do this. Take a look in [here](https://github.com/Financial-Times/authors-suggestion-api/blob/79e7bdaf87e02f18e1e959afc5c025617c2e4d7f/.circleci/config.yml#L63).

File: README.md [hunk 2]
```diff
@@ -30,24 +30,28 @@ Download the source code, dependencies and test dependencies:
...
+            --falcon-suggestion-endpoint           The endpoint for falcon suggestion api (env $FALCON_SUGGESTION_ENDPOINT) (default "/content/suggest/falcon")
+            --authors-suggestion-api-base-url      The base URL to authors suggestion api (env $AUTHORS_SUGGESTION_API_BASE_URL) (default "http://authors-suggestion-api:8080")
+            --authors-suggestion-endpoint          The endpoint for authors suggestion api (env $AUTHORS_SUGGESTION_ENDPOINT) (default "/content/suggest/authors")
+            --ontotext-suggestion-api-base-url     The base URL to ontotext suggestion api (env $ONTOTEXT_SUGGESTION_API_BASE_URL) (default "http://ontotext-suggestion-api:8080")
+            --ontotext-suggestion-endpoint         The endpoint for ontotext suggestion api (env $ONTOTEXT_SUGGESTION_ENDPOINT) (default "/content/suggest/ontotext")
+            --internal-concordances-api-base-url   The base URL for internal concordances api (env $CONCEPT_CONCORDANCES_API_BASE_URL) (default "http://internal-concordances:8080")
```
Comment (andreic92) [inline README.md]: I forgot to do this during broader story, but could you update this section please? Thanks 🙇

**Pull request merged**